### ISSUE TITLE
font: inherit - scales/font-family - crash

### ISSUE THREAD

**Issue by digitaljohn** [signal-noise/stylelint-scales#36]
The font-family rule breaks when using `font: inherit;`.

> Which rule, if any, is the bug related to?

`scales/font-family`

> What CSS is needed to reproduce the bug?

```css
a {
  font: inherit;
}
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "rules": {
    "scales/font-family": [
      ["XXX", "sans-serif"]
    ]
  }
}
```

> Which version of stylelint are you using?

e.g. `13.2.1`

> Which version of stylelint-scales are you using?

e.g. `1.2.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

e.g. "CLI with `stylelint 'src/{theme,style,components}/**/*.{js,jsx}'`"

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following warnings were flagged:

```shell
Checking formatting...
TypeError: family.forEach is not a function
    at /XXX/src/components/Chevron/styles.js:7:3
    at /XXX/node_modules/@signal-noise/stylelint-scales/lib/rules/font-family/index.js:35:14
```
